frontend/pages/profile.vue

```diff
@@ -30,6 +30,64 @@
       </ul>
     </div>
 
+    <!-- Statistics Section -->
+    <div class="statistics-section">
+      <div class="general-stats">
+        <h2>Allgemeine Statistiken</h2>
+        <ul>
+          <li>Beantwortete Item Konfigurationen: {{ stats.totalItemConfigs }}</li>
+          <li>Richtige Antworten: {{ stats.correctAnswers }}</li>
+          <li>Falsche Antworten: {{ stats.wrongAnswers }}</li>
+          <li>Durchgeführte Test Konfigurationen: {{ stats.totalTestConfigs }}</li>
+          <li>Versuche im Easy Modus: {{ stats.easyTries }}</li>
+          <li>Versuche im Medium Modus: {{ stats.mediumTries }}</li>
+          <li>Versuche im Hard Modus: {{ stats.hardTries }}</li>
+          <li>Test Konfigurationen mit freiem Modus: {{ stats.testConfigsFree }}</li>
+        </ul>
+      </div>
+      <div class="line-chart">
+        <h2>Scores nach Schwierigkeitsgrad</h2>
+        <Line :data="lineChartData" :options="lineChartOptions" />
+      </div>
+    </div>
+
+    <!-- Test Config Results as Expandable Cards -->
+    <div class="test-config-results">
+      <h2>Test Konfigurationsergebnisse</h2>
+
+      <!-- Ladeanimation anzeigen, wenn geladen wird -->
+      <div v-if="isLoadingTestConfigs" class="loader-container">
+        <div class="loader"></div>
+      </div>
+
+      <!-- Test Config Results Liste anzeigen, wenn das Laden abgeschlossen ist -->
+      <div v-else class="test-config-list">
+        <div
+          v-for="testConfig in testConfigResults"
+          :key="testConfig.id"
+          class="test-config-card"
+        >
+          <div class="card-header" @click="toggleCard(testConfig.id)">
+            <span class="card-title">Test ID: {{ testConfig.id }}</span>
```

Comment:
Small Thing: Id call it Sehtest ID and I would also add the name of it so its easier to find the right test for the user

Reply:
changed it